### PR TITLE
gl v4.5.3 may make installation easier for Windows

### PR DESCRIPTION
**Pull request by ATerentjev**
## Launch Checklist

 - [ ] briefly describe the changes in this PR
 - [ ] include before/after visuals or gifs if this PR includes visual changes
 - [ ] write tests for all new functionality
 - [ ] document any changes to public APIs
 - [ ] post benchmark scores
 - [ ] manually test the debug page
 - [ ] tagged `@mapbox/map-design-team` `@mapbox/static-apis` if this PR includes style spec API or visual changes
 - [ ] tagged `@mapbox/gl-native` if this PR includes shader changes or needs a native port
 - [ ] apply changelog label ('bug', 'feature', 'docs', etc) or use the label 'skip changelog'
 - [ ] add an entry inside this element for inclusion in the `mapbox-gl-js` changelog: `<changelog></changelog>`

### CODE REVIEW

File: src/util/tile_request_cache.js [hunk 1]
```diff
@@ -151,6 +151,7 @@ export function enforceCacheSizeLimit(limit: number) {
 
     sharedCache
         .then(cache => {
+            if (!cache) return;
```
Comment (dhritzkiv) [inline src/util/tile_request_cache.js:154]: Is this related to the Windows installation fix? If not, it should probably be in its own Pull Request :)

**Comment by dhritzkiv** [comment]
Some context to this PR. See stackgl/headless-gl#196

Basically, versions before v4.5.0 did not come with prebuilt Windows binaries. And versions before v4.5.3 came with missing dlls, rendering the prebuilt binaries useless for most (all?) Windows users. In the cases of missing/invalid prebuilt binaries, the user would have to have headless-gl be built on their machine, which either took several minutes to an hour, or was not possible if Visual Studio was not present.

In terms of changes since v4.3.3:
- there was a ES6 refactor, but is functionally identical
- introduction of linux musl prebuilds
- improved stencil call performance
- added support for OES_standard_derivatives extension

Overall, since this module is only used as a dev dependency, it's safe to upgrade.

**Comment by ryanhamley** [comment]
The change to `tile_request_cache.js` is the same as the change from https://github.com/mapbox/mapbox-gl-js/pull/9877 which I closed yesterday as being a band-aid and suggesting that it would be better to figure out the real cause here, since I'm not sure how `cache` would ever be undefined in that scenario. That being said, if others feel it's ok to accept the change since we don't really work with Blazor and probably aren't going to spend much time investigating the actual root cause, I'm fine with it.

File: src/util/tile_request_cache.js [hunk 2]
```diff
@@ -151,6 +151,7 @@ export function enforceCacheSizeLimit(limit: number) {
 
     sharedCache
         .then(cache => {
+
```
Comment (mourner) [inline src/util/tile_request_cache.js:154]: Accidental empty line.

File: package.json [hunk 3]
```diff
@@ -67,7 +67,7 @@
     "eslint-plugin-react": "^7.12.4",
     "esm": "~3.0.84",
     "flow-bin": "^0.100.0",
-    "gl": "~4.3.3",
+    "gl": "^4.5.3",
```
Comment (mourner) [inline package.json:70]: Still missing in `yarn.lock`.

**Comment by mourner** [comment]
Merged as #9906
